lib/public.js

```diff
@@ -66,15 +65,15 @@ function register (server, options, next) {
     }
   }, {
     method: 'GET',
-    path: '/hoodie/bundle.js',
+    path: '/hoodie/client.js',
     handler: {
-      file: path.join(client, 'dist/hoodie.js')
+      file: './.hoodie/client.js'
     }
   }, {
     method: 'GET',
-    path: '/hoodie/bundle.min.js',
+    path: '/hoodie/client.min.js',
     handler: {
-      file: path.join(client, 'dist/hoodie.min.js')
+      file: './.hoodie/client.min.js'
```

Comment:
@christophwitzko I think you once mentioned we should _always_ use `path.join`? Or is this okay like this?